### PR TITLE
Update remote-help.md

### PR DESCRIPTION
**Pull request by msbemba**
Changed the wording to reduce ambiguity

Fixes https://github.com/MicrosoftDocs/memdocs/issues/4025

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Review by Smritib17 — APPROVED** [review]

**Comment by Smritib17** [comment]
#sign-off

File: memdocs/intune/fundamentals/remote-help.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ The Remote Help app supports the following capabilities:
 - [Remote Help add on license or an Intune Suite license](intune-add-ons.md#available-add-ons) for all IT support workers (helpers) and users (sharers)
 - [Supported platforms and devices](#supported-platforms-and-devices)
 - The Remote Help app for Windows. See [Install and update Remote Help](#install-and-update-remote-help)
-- The helpers' device must be enrolled in Intune
+- The helper must be signed-in on an Intune Enrolled Device.
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/remote-help.md:82]:
```suggestion
- The helper must be signed in on an Intune enrolled device.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**